### PR TITLE
Reverted translations for HSM

### PR DESCRIPTION
**Pull request by DigneshGujarathi**

### CODE REVIEW

File: src/containers/Form/FormLayout.tsx [hunk 1]
```diff
@@ -98,6 +100,8 @@ export const FormLayout: React.SFC<FormLayoutProps> = ({
   const [action, setAction] = useState(false);
   const [link, setLink] = useState(undefined);
   const [deleted, setDeleted] = useState(false);
+  const [SaveClick, onSaveClick] = useState(false);
```
Comment (mdshamoon) [inline src/containers/Form/FormLayout.tsx:103]:
```suggestion
  const [saveClick, onSaveClick] = useState(false);
```

**Comment by mdshamoon** [comment]
One minor feedback

image

The sample message is still editable on the edit screen.

**Comment by mdshamoon** [comment]
Rest functionality is working fine.

**Comment by DigneshGujarathi** [comment]
> One minor feedback
> 
> image
> 
> The sample message is still editable on the edit screen.

@mdshamoon It looks like an issue with the EmojiInput component, Can we handle it as a separate issue?

**Comment by mdshamoon** [comment]
> @mdshamoon It looks like an issue with the EmojiInput component, Can we handle it as a separate issue?

Sure. Let's track it in a separate issue.

**Pull request merged**